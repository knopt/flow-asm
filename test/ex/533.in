17
3
-6.102918028067121
7.604420823085853 -7.442530167451165 -6.797084782808362 -6.145972751114441 4.26094487227207 1.7473354040083589 -9.266027791590082 -9.397732292095444 3.138538260011824 -8.524778993371452 -1.2405135404005918 -3.150559724798468 -2.398193043088317 -2.9642206577529535 3.146146880992802 -4.225580549242293 1.978092525455482 
6.426796623647867 5.4212450631865075 -8.776012712745283 9.515310923532894 -9.64213646998963 -4.7207628246770605 -9.776758791588842 -6.310695937766074 8.581337848940297 6.692694412478669 -9.69417840312852 -6.378354702191555 -1.224052294335202 2.8338584305174823 -9.302379325816927 7.608284895085895 4.309929492262141 
-1.8258831019753643 6.530408500278657 -6.317496540243104 2.6371437520032774 -4.306722145214373 -5.269882995740128 -0.00980671885920259 -5.545850743425129 -6.96115016350322 -7.864141320719085 -4.521285182161499 6.660574807613187 -1.5655084490140432 7.248565195580706 -7.928840926821941 1.849938032526948 9.495154511202674 

2.2924035552796695 2.09296694527324 -1.7282069442731824 
