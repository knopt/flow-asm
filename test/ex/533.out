80.8461219609853 -262.4227689096565 -65.34821505887416 -81.7009015789029 120.55591052171607 95.39026938176053 -101.10211066918892 -26.728276225869074 104.09731181981343 -276.97326978347223 46.39106727170932 44.826721541212926 19.318354743346685 -52.42394706461246 118.31542491608849 -90.45245276470555 -8.753011604077978 
151.05546810438165 101.92676497180574 -224.05403938585235 454.88172055524024 -340.24567080640287 -68.1505429310278 -201.1926408074021 8.596881476857188 423.4795545848632 281.8956442260872 -211.1690019647117 -128.10600709517644 3.119880501430786 94.82131954355654 -307.41409129098696 340.23037988951 33.87098779651172 
-76.70384116574357 64.9296394935751 -141.3636629218718 84.96324734108721 -98.47770504374355 12.184294445378073 120.44949445188354 -8.843201097665325 -114.42280562225308 -202.57960886573778 -20.988363127924046 254.2897859130395 -24.480093013617104 139.6904176711481 -157.8565860516336 94.44798421760872 99.31369783187664 
